ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/BoardPanel.java

```diff
@@ -37,16 +37,24 @@ public BoardPanel(BoardSize boardSize, SquareConfig squareConfig,
         mBoardSize = boardSize;
         mSquareLabels = new ArrayList<>(boardSize.width * boardSize.height);
 
-        GridLayout gridLayout = new GridLayout(mBoardSize.width + 2, mBoardSize.height + 2);
+        GridLayout gridLayout = new GridLayout(mBoardSize.height + 1, mBoardSize.width + 2);
```

Comment:
can we have:
`private static final int VERTICAL_PADDING = 1` and
`private static final int HORIZONTAL_PADDING = 2`
instead of having magic 1's and 2's everywhere?